cli/genkey/genkey_test.go

```diff
@@ -1 +1,45 @@
 package genkey
+
+import (
+	"encoding/json"
+	"github.com/cloudflare/cfssl/cli"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+func TestGenkey(t *testing.T) {
+	//testing through console
+	gopath := os.Getenv("GOPATH")
+	cfssl := path.Join(gopath, "bin", "cfssl")
+	testdata := path.Join(gopath, "src", "github.com", "cloudflare", "cfssl", "testdata")
+
+	out1, err := exec.Command(cfssl, "genkey", path.Join(testdata, "csr.json")).Output()
+	var response map[string]interface{}
+	json.Unmarshal(out1, &response)
```

Comment:
this is not satisfactory. Check the error and make sure the content of response is what you need. What key would response have? What value would be corresponding to each key? etc.